DigitalLearningSolutions.Web/Helpers/CustomPolicies.cs

```diff
@@ -18,6 +19,13 @@ public static AuthorizationPolicyBuilder ConfigurePolicyBasicUser(AuthorizationP
             return policy.RequireAssertion(context => context.User.GetUserId() != null);
         }
 
+        public static AuthorizationPolicyBuilder ConfigurePolicyCentreUser(AuthorizationPolicyBuilder policy)
+        {
+            return policy.RequireAssertion(
+                context => context.User.GetUserId() != null && context.User.GetCentreId() != null

```

Comment:
We've got a number of helper methods defined in the CustomClaimHelper such as `HasCentreAdminPermissions`. Do we want to define similar ones for the IsBasicUser and IsCentreUser scenarios, and apply them across these policies?

Reply:
I'm not sure that it adds much? I don't think these levels will ever be used outside policies, unlike the HasCentreAdminPermissions/HasCentreManagerPermissions extension methods.

Reply:
Fair enough.